### PR TITLE
Add a blueberry buttermilk pancakes recipe

### PR DESCRIPTION
**Pull request by githubstudent**

### CODE REVIEW

**Comment by githubstudent** [comment]
@githubteacher any comments here? I’d like to add this pancakes recipe.

File: _posts/pancakes.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Blueberry Buttermilk Pancakes
```
Comment (githubteacher) [inline _posts/pancakes.md]: You need to include the meta-data so that this file appears in the recipes view.

**Comment by githubteacher** [comment]
Maybe also include a photo? That always works.

**Comment by githubstudent** [comment]
@githubteacher I think I did everything, can this be merged?

**Pull request merged**